contracts/conduit/ConduitController.sol

```diff
@@ -95,11 +95,13 @@ contract ConduitController is ConduitControllerInterface {
         // Deploy the conduit via CREATE2 using the conduit key as the salt.
         new Conduit{ salt: conduitKey }();
 
+        ConduitProperties storage conduitProperties = _conduits[conduit];
```

Comment:
Mind adding a comment above this line? Something like "Initialize storage variable referencing conduit properties."